Sources/TestsCommon/ServerVersion.swift

```diff
@@ -5,6 +5,7 @@ public struct ServerVersion: Comparable, Decodable, CustomStringConvertible {
     public static let mongodFailCommandSupport = ServerVersion(major: 4, minor: 0)
     public static let mongosFailCommandSupport = ServerVersion(major: 4, minor: 1, patch: 5)
     public static let mongodBlockTimeSupport = ServerVersion(major: 4, minor: 2, patch: 9)
+    public static let mongodHintSupport = ServerVersion(major: 4, minor: 4)
```

Comment:
see my comment on `supportsHint`, but I think we'll want to just use this directly in the test without defining a constant for it. Also note that hints have had gained support in different methods over a few different versions of MongoDB--4.4 was just the one for `findAndModify`.

Reply:
removed